src/Equations/damaged-elastic/Kernels/Time.cpp

```diff
@@ -703,3 +700,1491 @@ void seissol::kernels::Time::flopsTaylorExpansion(long long& nonZeroFlops,
+                  2 * mu_eff * (dofsNPlus[2 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitzz);
+
+    real sxy_sp = 2 * mu_eff * (dofsNPlus[3 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitxy);
+    real syz_sp = 2 * mu_eff * (dofsNPlus[4 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInityz);
+    real szx_sp = 2 * mu_eff * (dofsNPlus[5 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitzx);
+
+    // breakage stress
+    real sxx_bp = (2.0 * aB2 + 3.0 * xip * aB3) * EspIp + aB1 * std::sqrt(EspIIp) +
+                  (2.0 * aB0 + aB1 * xip - aB3 * xip * xip * xip) *
+                      (dofsNPlus[0 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitxx);
+    real syy_bp = (2.0 * aB2 + 3.0 * xip * aB3) * EspIp + aB1 * std::sqrt(EspIIp) +
+                  (2.0 * aB0 + aB1 * xip - aB3 * xip * xip * xip) *
+                      (dofsNPlus[1 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInityy);
+    real szz_bp = (2.0 * aB2 + 3.0 * xip * aB3) * EspIp + aB1 * std::sqrt(EspIIp) +
+                  (2.0 * aB0 + aB1 * xip - aB3 * xip * xip * xip) *
+                      (dofsNPlus[2 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitzz);
+
+    real sxy_bp = (2.0 * aB0 + aB1 * xip - aB3 * xip * xip * xip) *
+                  (dofsNPlus[3 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitxy);
+    real syz_bp = (2.0 * aB0 + aB1 * xip - aB3 * xip * xip * xip) *
+                  (dofsNPlus[4 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInityz);
+    real szx_bp = (2.0 * aB0 + aB1 * xip - aB3 * xip * xip * xip) *
+                  (dofsNPlus[5 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitzx);
+
+    dofsStressNPlus[0 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * sxx_sp +
+        dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * sxx_bp;
+
+    dofsStressNPlus[1 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * syy_sp +
+        dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * syy_bp;
+
+    dofsStressNPlus[2 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * szz_sp +
+        dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * szz_bp;
+
+    dofsStressNPlus[3 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * sxy_sp +
+        dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * sxy_bp;
+
+    dofsStressNPlus[4 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * syz_sp +
+        dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * syz_bp;
+
+    dofsStressNPlus[5 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * szx_sp +
+        dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * szx_bp;
```

Comment:
Duplicate to lines 1048 ff.

Reply:
Put them in a function, but the function definition is heavy..